src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -1808,7 +1808,9 @@ private List<CompletionResult> GetResultForIdentifier(CompletionContext completi
                             case UsingStatementKind.Type:
                                 break;
                             default:
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                                 throw new ArgumentOutOfRangeException("UsingStatementKind");
+#pragma warning restore CA2208 // https://github.com/PowerShell/PowerShell/issues/13909
```

Comment:
Why not use nameof?

Reply:
I decided to suppress because `UsingStatementKind` is not the name of a method parameter, therefore this would arguably be incorrect use.